issue_processing/jira_issue_processing.py

```diff
@@ -645,7 +676,7 @@ def print_to_disk_bugs(issues, results_folder):
                 ))
 
     # write to output file
-    csv_writer.write_to_csv(output_file, lines)
+    csv_writer.write_to_csv(output_file, lines, True)
```

Comment:
Please add the name for the last parameter as it is not self-explaining: `append=True`.

This also holds for all other occurrences below.